pkgs/games/xtris/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchzip, xorg }:
+stdenv.mkDerivation rec {
+  name = "xtris-${version}";
+  version = "1.15";
+
+  src = fetchzip {
+    url = "https://web.archive.org/web/20120315061213/http://www.iagora.com/~espel/xtris/xtris-${version}.tar.gz";
+    sha256 = "1vqva99lyv7r6f9c7yikk8ahcfh9aq3clvwm4pz964wlbr9mj1v6";
+  };
+
+  patchPhase = ''
+    sed -i '
+      s:/usr/local/bin:'$out'/bin:
+      s:/usr/local/man:'$out'/share/man:
+      s:mkdir:mkdir -p:g
+      s:^CFLAGS:#CFLAGS:
+    ' Makefile
+  '';
+  buildInputs = [ xorg.libX11 ];
+
+  meta = with stdenv.lib; {
+    description = "xtris is a multi-player version of the classical game of Tetris, for the X Window system";
```

Comment:
`description` shouldn't contain the program name. Ie. `A multi-player version of the classical game of Tetris, for the X Window system`. See https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.

Reply:
done